frontend/styles/components/calendar/DailyView.module.scss

```
@@ -67,20 +67,12 @@
   height: 100%;
   width: 4px;
   background-color: $brand-pink-color;
-  z-index: 10; 
+  z-index: 10;
+  // Purely a "now" marker, not interactive -- without this it sits on top of whatever
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)